salt/modules/zypper.py

```diff
@@ -665,23 +665,20 @@ def install(name=None,
                        'new': '<new-version>'}}
     '''
     try:
-        pkg_params, pkg_type = __salt__['pkg_resource.parse_targets'](
-            name, pkgs, sources, **kwargs
-        )
+        pkg_params, pkg_type = __salt__['pkg_resource.parse_targets'](name, pkgs, sources, **kwargs)
```

Comment:
Any reason this was changed? I'm not a fan of moving stuff around just to move it around.